src/niscope/metadata/functions_addon.py

```diff
@@ -61,11 +61,11 @@
     'Fetch':                            { 'codegen_method': 'private',  },
     'Read':                             { 'codegen_method': 'private',  },
     'ActualNumWfms':                    { 'codegen_method': 'private',  },  # We use it internally so the customer doesn't have to.
-    '.etAttributeViInt64':              { 'codegen_method': 'no',       },  # NI-SCOPE has no ViInt64 attributes.
-    'ClearWaveformProcessing':          { 'codegen_method': 'no',       },  # Per #667, removing waveform measurement methods
-    'AddWaveformProcessing':            { 'codegen_method': 'no',       },  # Per #667, removing waveform measurement methods
-    'FetchArrayMeasurement':            { 'codegen_method': 'no',       },  # Per #667, removing waveform measurement methods
-    'ActualMeasWfmSize':                { 'codegen_method': 'no',       },  # Per #667, removing waveform measurement methods
+    '.etAttributeViInt64':              { 'codegen_method': 'private',  },  # NI-SCOPE has no ViInt64 attributes.
```

Comment:
has no *public** ViInt ...

Does this work though? The Attribute classes don't know to add the underscore.

Reply:
Line removed. We already make all Get/SetAttribute functions private with an entry earlier in the dictionary. All the Attrubute classes already use private functions.